src/vtab/data_chunk.rs

```diff
@@ -59,6 +59,11 @@ impl DataChunk {
     pub fn num_columns(&self) -> usize {
         unsafe { duckdb_data_chunk_get_column_count(self.ptr) as usize }
     }
+
+      /// Get the ptr of duckdb_data_chunk in this [DataChunk].
+      pub fn get_ptr(&self) -> duckdb_data_chunk {
+        self.ptr.clone()
```

Comment:
why need clone

Reply:
https://github.com/web3creator/duckdb-rs/blob/2078ebcabf7c44e785da49471b40761e0b073c5e/src/appender.rs#L102

This pointer is needed here

Reply:
https://github.com/web3creator/duckdb-rs/blob/2078ebcabf7c44e785da49471b40761e0b073c5e/src/appender.rs#L102

This pointer is needed here